app/assets/stylesheets/forever_style_guide/globals/_variables.scss

```diff
@@ -37,6 +37,11 @@ $btn-padding-large-horizontal: $padding-large-horizontal * 1.5;
 $btn-padding-small-horizontal: $padding-small-horizontal * 1.5;
 $btn-padding-xs-horizontal: $padding-xs-horizontal * 1.5;
 
+$btn-padding-default-vertical : $padding-default-vertical - 2;
```

Comment:
Looks like these variables are fairly localized, so it seems reasonable to change. Only question I had was if it was worth keeping the variable names as is since it mimics the padding-sm-vertical (padding-*-*) pattern that we are taking from the bootstrap vars?